xml/chapter5/section5/subsection3.xml

```diff
@@ -434,14 +434,15 @@ function construct_arglist(arg_codes) {
 }
 
 function code_to_get_rest_args(arg_codes) {
-    const code_for_next_arg = preserving(
-        list("argl"), 
-        head(arg_codes),
-        make_instruction_sequence(
-            list("val", "argl"), 
-            list("argl"),
-            list(assign("argl", list(op("pair"), 
-                                     reg("val"), reg("argl"))))));
+    const code_for_next_arg = 
```

Comment:
Let's go with this reformat for now -- but (I'm just echoing Julie here) we will have quite a bit of horizontal space so vertical space is a more scarce resource. Again, expecting to have to revisit all formatting of code when typesetting the book.